custom/icds_reports/templates/icds_reports/icds_app/download.directive.html

```diff
@@ -127,6 +156,19 @@
                         </ui-select>
                     </div>
                 </div>
+                <div class="form-group" ng-if="($ctrl.showViewBy() && $ctrl.selectedIndicator != 10 && $ctrl.selectedIndicator == 15)">
```

Comment:
Hey, instead of creating  a new filter which behaves like the existing one could be confusing latest.
I think better would be use the existing one only.
To remove the location hierarchy below district you could do it manually on the places where its options are updated.

Currently its options variable groupByLevels
if modified in two places.
[During Init](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/directives/download/download.directive.js#L224)
and [During Location select](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/directives/download/download.directive.js#L260)

here is the service is called in the above two places: https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/static/js/angular-services/locations.service.js#L238

remove the block, sector, awc levels from groupByLevels on above two places based on report selected should be easy and wont need us to create a new filter.